common/helpers/qiqo_chat.py

```diff
@@ -25,15 +25,17 @@ def __init__(self, contributor):
         # Throttle user creation calls to qiqochat
         log_result = 'Subscribe attempt to qiqochat for {user} at {time}'\
             .format(time=timezone.now(), user=self.user.id_full_name())
-        if not self.user.qiqo_signup_time \
-                or (timezone.now() - self.user.qiqo_signup_time).total_seconds() > settings.QIQO_SIGNUP_TIMEOUT_SECONDS:
+        if self.user.qiqo_signup_time \
+                and (timezone.now() - self.user.qiqo_signup_time).total_seconds() <= settings.QIQO_SIGNUP_TIMEOUT_SECONDS:
+            log_result += ':Aborting, last request < {time}s ago'.format(time=settings.QIQO_SIGNUP_TIMEOUT_SECONDS)
+        elif self.user.full_name() == ' ':
```

Comment:
```suggestion
        elif self.user.full_name() == None or self.user.full_name() == "" or self.user.full_name().isspace():
```
We want to cover all the possible bases of the blank name case.  Would recommend making this into a python helper method, similar to the javascript equivalent we use: https://github.com/DemocracyLab/CivicTechExchange/blob/b7d71d0c3ea144f71f904b6a8490abe2afcf0f84/common/components/utils/string.js#L9